python/lsst/meas/astrom/pessimistic_pattern_matcher_b_3D.py

```diff
@@ -429,8 +445,8 @@ def _compute_test_vectors(self, source_array):
         ztop_vect /= np.sqrt(np.dot(ztop_vect, ztop_vect))
 
         # Return our list of vectors for later rotation testing.
-        return [xbtm_vect, xtop_vect, ybtm_vect, ytop_vect,
-                zbtm_vect, ztop_vect]
+        return np.array([xbtm_vect, xtop_vect, ybtm_vect, ytop_vect,
+                         zbtm_vect, ztop_vect])
```

Comment:
Why are we changing from a `list` to an `np.array`? May be for good reasons, but it makes the docstring on this function (which asserts that it returns a `list`) wrong.

Reply:
I've changed the comment to reflect the new output. The output is now array to make it more closely reflect the data format of the input source and reference objects to be matched.